packages/syft/src/syft/service/worker/worker_pool_service.py

```diff
@@ -179,3 +177,96 @@ def delete_worker(
         return SyftSuccess(
             message=f"Worker with id: {worker_id} deleted successfully from pool: {worker_pool.name}"
         )
+
+    @service_method(
+        path="worker_pool.worker_logs",
+        name="worker_logs",
+        roles=DATA_OWNER_ROLE_LEVEL,
+    )
+    def worker_logs(
+        self,
+        context: AuthedServiceContext,
+        worker_pool_id: UID,
+        worker_id: UID,
+    ) -> Union[bytes, SyftError]:
+        worker_pool_worker = self._get_worker_pool_and_worker(
+            context, worker_pool_id, worker_id
+        )
+        if isinstance(worker_pool_worker, SyftError):
+            return worker_pool_worker
+
+        _, worker = worker_pool_worker
+
+        docker_container = _get_worker_container(worker)
+        if isinstance(docker_container, SyftError):
+            return docker_container
+
+        try:
+            return docker_container.logs()
```

Comment:
This returns `bytes`, not `str`. I opted to just return it as is here. Users can `decode()` to string if needed